### PR TITLE
populate and use tags on executions.

### PR DESCRIPTION
**Pull request by jdhollen**
**Related issues**: N/A

### CODE REVIEW

**Comment by jdhollen** [comment]
lemme know if you want me to split this up, it's just so... small

**Comment by jdhollen** [comment]
just kidding, since the related code on using tags hasn't rolled out yet it'd be a pain to push the schema update, so i'm going to split that part off.

**Comment by jdhollen** [comment]
alright, this is caught up and ready now.

**Review by tylerwilliams — APPROVED** [review]

File: server/util/clickhouse/clickhouse.go [hunk 1]
```diff
@@ -179,7 +180,7 @@ func buildExecution(in *repb.StoredExecution, inv *sipb.StoredInvocation) *schem
 		Command:                            inv.GetCommand(),
 		Success:                            inv.GetSuccess(),
 		InvocationStatus:                   inv.GetInvocationStatus(),
-		Tags:                               []string{},
+		Tags:                               invocation_format.ConvertDbTagsToOlap(inv.GetTags()),
```
Comment (luluz66) [inline server/util/clickhouse/clickhouse.go:183]: nit: not this PR, but the function name should be ConvertDBTagsToOLAP for go conventions.

**Pull request merged**